packages/image_picker/ios/Classes/ImagePickerPlugin.m

```diff
@@ -253,6 +253,10 @@ - (void)imagePickerController:(UIImagePickerController *)picker
     NSNumber *maxHeight = [_arguments objectForKey:@"maxHeight"];
     NSNumber *imageQuality = [_arguments objectForKey:@"imageQuality"];
 
+    if (![imageQuality isKindOfClass:[NSNumber class]]) {
+      imageQuality = @1;
+    }
+      
     if (imageQuality.intValue < 0 || imageQuality.intValue > 100) {
```

Comment:
make it `else if`. in iOS nil.intValue will return 0 which is not what we wanted.

Reply:
done